data/pathfinder/paizo/player_companion/healers_handbook/hhb_spells.lst

```diff
@@ -27,6 +27,6 @@ Befuddled Combatant			CLASSES:Bard,Mesmerist,Psychic,Shaman,Sorcerer,Wizard,Witc
 # Bardic Masterpiece
 # ==============================
 
-Masterpiece (Endless Waltz of the Eldest)				CLASSES:Bard=2	PREMULT:2,[PRESKILL:1,Perform (Dance)=4,Perform (String Instruments)=4]		SCHOOL:Masterpiece	CASTTIME:1 standard action		SOURCEPAGE:19
+Masterpiece (Endless Waltz of the Eldest)				CLASSES:Bard=2	PRESKILL:1,Perform (Dance)=4,Perform (String Instruments)=4		SCHOOL:Masterpiece	CASTTIME:1 standard action		SOURCEPAGE:19
```

Comment:
Looks like you changed the PREREQs which shouldn't have to do with this change. Were these coded incorrectly?

Reply:
Yes, It was a bad C/P error. The prereqs were 4 dance OR 4 string, not 4 of both